Geometry/TrackerGeometryBuilder/python/trackerDigiGeometryESModule_cfi.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+## TEMPLATE as it used in the HLT menu
+trackerDigiGeometryESModule = cms.ESProducer( "TrackerDigiGeometryESModule",
+  appendToDataLabel = cms.string( "" ),
+  fromDDD = cms.bool( False ),
+  trackerGeometryConstants = cms.PSet(
+    ROCS_X = cms.int32( 0 ),
+    ROCS_Y = cms.int32( 0 ),
+    upgradeGeometry = cms.bool( False ),
+    BIG_PIX_PER_ROC_Y = cms.int32( 2 ),
+    BIG_PIX_PER_ROC_X = cms.int32( 1 ),
+    ROWS_PER_ROC = cms.int32( 80 ),
+    COLS_PER_ROC = cms.int32( 52 )
+  ),
+  applyAlignment = cms.bool( True ),
+  alignmentsLabel = cms.string( "" )
+)
```

Comment:
Thou shalt not introduce cfi files, if they are automatically produced by
fillDescriptions.

.Andrea
On 16 Jan 2015 19:10, "mia tosi" <notifications@github.com> wrote:

> In
> Geometry/TrackerGeometryBuilder/python/trackerDigiGeometryESModule_cfi.py
> <https://github.com/cms-sw/cmssw/pull/7193#discussion-diff-23097637>:
>
> > +## TEMPLATE as it used in the HLT menu
> > +trackerDigiGeometryESModule = cms.ESProducer( "TrackerDigiGeometryESModule",
> > +  appendToDataLabel = cms.string( "" ),
> > +  fromDDD = cms.bool( False ),
> > +  trackerGeometryConstants = cms.PSet(
> > +    ROCS_X = cms.int32( 0 ),
> > +    ROCS_Y = cms.int32( 0 ),
> > +    upgradeGeometry = cms.bool( False ),
> > +    BIG_PIX_PER_ROC_Y = cms.int32( 2 ),
> > +    BIG_PIX_PER_ROC_X = cms.int32( 1 ),
> > +    ROWS_PER_ROC = cms.int32( 80 ),
> > +    COLS_PER_ROC = cms.int32( 52 )
> > +  ),
> > +  applyAlignment = cms.bool( True ),
> > +  alignmentsLabel = cms.string( "" )
> > +)
>
> yes, but @gennai <https://github.com/gennai> asked for having a physical
> file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/7193/files#r23097637>.
>